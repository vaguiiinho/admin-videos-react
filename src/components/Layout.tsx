import { ReactNode } from "react";
import { Box, Container } from "@mui/material";

export function Layout({ children }: { children: ReactNode }) {
    return (
        <Box>
            <Container maxWidth="lg" sx={{ my: 4, bg: "white", backgroundColor: '#000' }}>
                {children}
            </Container>
        </Box>
    )
}
