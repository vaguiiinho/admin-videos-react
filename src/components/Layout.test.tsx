import { render } from "@testing-library/react";
import { Layout } from "./Layout";

describe("Layout", () => {
    it("should render correctly", () => {
        const { asFragment } = render(
            <Layout >
                <h1>Hello World</h1>
            </Layout>
        );
        expect(asFragment()).toMatchSnapshot();
    });
});