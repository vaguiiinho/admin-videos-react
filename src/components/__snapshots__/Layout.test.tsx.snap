// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should render correctly 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-0"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1g3nlq1-MuiContainer-root"
    >
      <h1>
        Hello World
      </h1>
    </div>
  </div>
</DocumentFragment>
`;
